### PR TITLE
library upgrade and python version upgrade

### PR DESCRIPTION
**Pull request by Buhle79**
Celery upgrade

### CODE REVIEW

File: eventstore/test_hcs_tasks.py [hunk 1]
```diff
@@ -12,7 +12,7 @@ class UpdateTurnContactTaskTest(TestCase):
     def test_update_turn_contact_task(self):
         responses.add(
             responses.PATCH,
-            f"https://turn/v1/contacts/27820001001/profile",
+            f"{'https://turn/v1/contacts/27820001001/profile'}",
```
Comment (erikh360) [inline eventstore/test_hcs_tasks.py]: for these we can just remove the `f` since we're not replacing anything

File: eventstore/test_hcs_tasks.py [hunk 2]
```diff
@@ -22,6 +22,6 @@ def test_update_turn_contact_task(self):
         [patch] = responses.calls
 
         self.assertEqual(
-            patch.request.url, f"https://turn/v1/contacts/27820001001/profile"
+            patch.request.url, f"{'https://turn/v1/contacts/27820001001/profile'}"
```
Comment (erikh360) [inline eventstore/test_hcs_tasks.py]: same as above, so just `"https://turn/v1/contacts/27820001001/profile"` no f or {}

File: eventstore/test_tasks.py [hunk 3]
```diff
@@ -91,7 +91,7 @@ def test_conversation_expired(self):
         )
 
         responses.add(
-            responses.POST, f"http://turn/v1/chats/27820001001/archive", json={}
+            responses.POST, f"{'http://turn/v1/chats/27820001001/archive'}", json={}
```
Comment (erikh360) [inline eventstore/test_tasks.py]: same

File: eventstore/test_tasks.py [hunk 4]
```diff
@@ -111,7 +111,7 @@ def test_conversation_expired(self):
             json.loads(turn_archive.request.body),
             {
                 "before": "ABGGJ4NjeFMfAgo-sCqKaSQU4UzP",
-                "reason": f"Auto archived after 11 days",
+                "reason": f"{'Auto archived after 11 days'}",
```
Comment (erikh360) [inline eventstore/test_tasks.py]: same

File: ndoh_hub/settings.py [hunk 5]
```diff
@@ -139,7 +139,7 @@
 
 USE_L10N = True
 
-USE_TZ = True
+# USE_TZ = True
```
Comment (erikh360) [inline ndoh_hub/settings.py]: Not sure if we need this change?

File: ndoh_hub/testsettings.py [hunk 6]
```diff
@@ -1,4 +1,4 @@
-from ndoh_hub.settings import *  # flake8: noqa
+from ndoh_hub.settings import REST_FRAMEWORK  # flake8: noqa
```
Comment (erikh360) [inline ndoh_hub/testsettings.py]: undo

File: setup.py [hunk 7]
```diff
@@ -16,10 +16,11 @@
         "Markdown==3.1.1",
         "dj-database-url==0.5.0",
         "django-environ==0.4.5",
-        "psycopg2==2.7.5",
+        # "psycopg2==2.7.5",
```
Comment (erikh360) [inline setup.py]: remove commented out line

**Review by erikh360 — COMMENTED** [review]
looks good, some small comments

File: eventstore/serializers.py [hunk 8]
```diff
@@ -397,3 +424,28 @@ class Code(serializers.Serializer):
 
 class ForgetContactSerializer(serializers.Serializer):
     contact_id = serializers.UUIDField(required=True)
+
+
+class RegistrationSerializer(serializers.ModelSerializer):
```
Comment (erikh360) [inline eventstore/serializers.py]: Should we not leave this one in the registration app? its not used in eventstore

File: eventstore/admin.py [hunk 9]
```diff
@@ -233,3 +236,27 @@ def get_readonly_fields(self, request, obj=None):
         if obj is None:
             return ()
         return super().get_readonly_fields(request, obj)
+
+
+class RegistrationAdmin(admin.ModelAdmin):
```
Comment (erikh360) [inline eventstore/admin.py]: I don't think this should be here?

File: eventstore/tasks.py [hunk 10]
```diff
@@ -111,6 +160,58 @@ def get_rapidpro_contact_by_msisdn(context, field):
     return context
 
 
+@app.task()
+def remove_personally_identifiable_fields(registration_id):
```
Comment (erikh360) [inline eventstore/tasks.py]: Don't need this

File: eventstore/tasks.py [hunk 11]
```diff
@@ -52,6 +96,11 @@ def get_utc_now():
     return datetime.now(tz=pytz.utc)
 
 
+is_client = IdentityStoreApiClient(
```
Comment (erikh360) [inline eventstore/tasks.py]: Don't need this

File: scripts/migrate_to_rapidpro/fix_auto_baby_switch_hub.py [hunk 12]
```diff
@@ -19,7 +19,7 @@ async def process_babyswitch(session, row, target, jembi_writer):
     contact_id = row["contact_id"]
     timestamp = row["timestamp"]
 
-    url = urllib_parse.urljoin(HUB_URL, f"/api/v2/babyswitches/")
+    url = urllib_parse.urljoin(HUB_URL, f"{'/api/v2/babyswitches/'}")
```
Comment (erikh360) [inline scripts/migrate_to_rapidpro/fix_auto_baby_switch_hub.py]: Don't need the F and {''}

**Review by erikh360 — COMMENTED** [review]
Looking good, just a few things I think we don't need

File: setup.py [hunk 13]
```diff
@@ -44,7 +44,7 @@
         "Framework :: Django",
         "License :: OSI Approved :: BSD License",
         "Programming Language :: Python",
-        "Programming Language :: Python :: 3.6",
+        "Programming Language :: Python :: 3.9.9",
```
Comment (rudigiesler) [inline setup.py:47]: I don't think there's a 3.9.9 classifier, only 3.9: https://pypi.org/classifiers/

File: eventstore/tasks.py [hunk 14]
```diff
@@ -618,3 +661,129 @@ def get_random_contact():
...
+        pass
+    for message_type in ("image", "document", "audio", "video", "voice", "sticker"):
+        try:
+            return message[message_type].get("caption", "<{}>".format(message_type))
+        except KeyError:
+            pass
+    try:
+        assert "contacts" in message
+        return "<contacts>"
+    except AssertionError:
```
Comment (rudigiesler) [inline eventstore/tasks.py]: This is a silly pattern that I used a while ago, might be a good opportunity to change it.

Using try/except is a good python pattern, but using `assert` in anything other than unit tests has issues, most notably that they can be disabled: https://docs.python.org/3/reference/simple_stmts.html#the-assert-statement

A better pattern here would be just an if statement.

**Review by rudigiesler — COMMENTED** [review]
Just two small comments, but overall looks great!

**Review by rudigiesler — APPROVED** [review]

**Review by erikh360 — APPROVED** [review]

**Pull request merged**